moped-etl/prefect/moped_test_create_flow.py

```diff
@@ -239,55 +249,80 @@
 
     basename = Parameter("basename")
 
-    remove_activity_log_sqs = remove_activity_log_sqs(basename=basename)
-    remove_activity_log_lambda = remove_activity_log_lambda(
+    remove_activity_log_sqs = activity_log.remove_activity_log_sqs(basename=basename)
+    remove_activity_log_lambda = activity_log.remove_activity_log_lambda(
         basename=basename, upstream_tasks=[remove_activity_log_sqs]
     )
 
+with Flow("Apply Database Migrations") as apply_database_migrations:
 
+    basename = Parameter("basename")
 
-if __name__ == "__main__":
-    print("main()")
-
-    basename = "netlify-test-deployment"
-    database = basename.replace("-", "_")
-    database_data_stage = "staging"
-
-    # flow execution is serialized!
-
-
-    # print("\n🍄 Decomissioning Database\n")
-    # database_decommission.run(basename=database)
-    # print("\n🍄 Comissioning Database\n")
-    # database_commission.run(basename=database)
-
-    # print("\n🚀 Decomissioning API\n")
-    # api_decommission.run(parameters=dict(basename=basename))
-    # print("\n️🚀 Comissioning API\n")
-    # api_commission.run(parameters=dict(basename=basename, database=database))
-
-    # print("\n🍄 Decomissioning Database\n")
-    # database_decommission.run(basename=database)
-    # print("\n🍄 Comissioning Database\n")
-    # database_commission.run(basename=database, stage=database_data_stage)
+    graphql_endpoint = "https://" + ecs.get_graphql_engine_hostname(basename=basename)
+    access_key = ecs.get_graphql_engine_access_key(basename=basename)
 
+    rm_clone = "rm -fr /tmp/atd-moped"
+    cleaned = ecs.shell_task(command=rm_clone)
+    git_clone = "git clone " + GIT_REPOSITORY + " /tmp/atd-moped"
+    cloned = ecs.shell_task(command=git_clone, upstream_tasks=[cleaned])
+    checkout_branch = "git -C /tmp/atd-moped/ checkout " + basename
+    checked_out = ecs.shell_task(command=checkout_branch, upstream_tasks=[cloned])
 
-    # print("\n🤖 Decomissioning ECS\n")
-    # ecs_decommission.run(parameters=dict(basename=basename))
-    # print("\n🤖 Comissioning ECS\n")
-    # ecs_commission.run(parameters=dict(basename=basename, database=database))
+    metadata = "metadata"
 
-    # print("💡 Comissioning Netlify Build & Deploy\n")
-    # netlify_commission.run(parameters=dict(basename=basename))
+    config = ecs.create_graphql_engine_config_contents(
+        graphql_endpoint=graphql_endpoint, access_key=access_key, metadata=metadata
+    )
```

Comment:
This section of the diff is very mangled - check the final file for something easier to grok: https://github.com/cityofaustin/atd-moped/blob/integrate-flows/moped-etl/prefect/moped_test_create_flow.py#L257